packages/dashboard/src/components/iot-resizable-panes/iot-resizable-panes.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { newSpecPage, jestSetupTestFramework } from '@stencil/core/testing';
+jestSetupTestFramework();
+import { IotResizablePanes } from './iot-resizable-panes';
+
+const leftSlotContent = `Warp core monitor`;
+const centerSlotContent = `Forward viewscreen`;
+const rightSlotContent = `Phaser controls`;
+
+const components = [IotResizablePanes];
+const html = `
+  <iot-resizable-panes>
+    <div slot="left">
+      <p>${leftSlotContent}</p>
+    </div>
+    <div slot="center">
+      <p>${centerSlotContent}</p>
+    </div>
+    <div slot="right">
+      <p>${rightSlotContent}</p>
+    </div>
+  </iot-resizable-panes>
+`;
+
+describe('iot-resizable-panes', () => {
```

Comment:
nit: we can remove this describe block, already in context of `iot-resizeable-panes` file

Reply:
Will do in follow-up PR